sql/srb2_highscores_init.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE `highscores` (
+  `username` varchar(30) COLLATE utf8mb4_unicode_ci NOT NULL,
+  `skin` varchar(20) COLLATE utf8mb4_unicode_ci NOT NULL,
+  `map_id` int(11) NOT NULL,
+  `time` int(11) DEFAULT NULL,
+  `time_string` text COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+  PRIMARY KEY (`username`,`skin`,`map_id`),
+  KEY `map_id` (`map_id`),
+  CONSTRAINT `highscores_ibfk_1` FOREIGN KEY (`map_id`) REFERENCES `maps` (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `maps` (
+  `id` int(11) NOT NULL,
+  `name` text COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+  `image` blob,
+  `votes` int(11) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+create table `voted` (
+  `ip` varchar(15) COLLATE utf8mb4_unicode_ci NOT NULL,
+  `map` int(11),
+  PRIMARY KEY (`ip`, `map`),
+  FOREIGN KEY (map) REFERENCES maps(id)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
```

Comment:
With SQLAlchemy the database should be initialized with `db.create_all()`, instead of a raw sql script.
Sorry for not including that part already.

I think the best way would be creating an `initialize.py` which makes a connection to the database and initializes it using the defined models.